_posts/2019-05-30-bitcoin-sov-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+To make my position clear about the block size, I absolutely advocate for small blocks. I believe that running a Bitcoin full node should be as easy and affordable as possible so the system is not only decentralized but fully distributed where final users have full sovereignty over their Bitcoins.
+
+Going back to the principles of the Austrian school, the most widely followed monetary theorist of this school, Ludwig Von Mises, developed the [Regression Theorem](https://wiki.mises.org/wiki/Regression_theorem) in his book titled ["The Theory of Money and Credit"](https://mises.org/library/theory-money-and-credit). The theorem states that for an economic good to be money it **must** primarily have other non-monetary uses before it can become money. Bitcoin does not fit this description. Some economists like [Block and Davidson](https://mises.org/library/bitcoin-regression-theorem-and-emergence-new-medium-exchange) have tried to reinterpret this theorem in order to shoehorn Bitcoin into it by claiming that the Regression Theorem applies only to barter economies and does not apply when there are already monetary prices. I don't think that interpretation is correct and it is also incoherent in and of itself (see [Juan Ramón Rallo rebuttal](https://juanramonrallo.com/mises-y-block-se-equivocan-sobre-sus-teoremas-regresivos-del-dinero/)).
+
+Mises' Regression Theorem is based on the observations made by Carl Menger in his seminal work ["On the Origins of Money"](https://mises-media.s3.amazonaws.com/On%20the%20Origins%20of%20Money_5.pdf) where he describes the process of how money arose from commodities. But, what Menger did was make a historical observation, not a theoretical explanation. It was Mises who wrongly took those observations into theory with his unfortunate Regression Theorem.
+
+In an attempt to dodge the Regression Theorem debate others have retorted with Nick Szabo's views in his remarkable essay ["Shelling out: The Origins of Money"](https://nakamotoinstitute.org/shelling-out/), using the collectible and store of value arguments. Szabo's excellent work is also a historical explanation like Menger, but more detailed and precise from an anthropological point of view.
+
+I shall not spend much time rebutting the claim that Bitcoin's source of value at the very beginning was being a collectible in the literal meaning of the term. That was not what Szabo meant and it is not credible at all to say that the first Bitcoin owners demanded it because it was a whimsy, rather than demanding it because they thought it could potentially become a medium of exchange. Bitcoin's first owners knew extremely well why Bitcoin was invented for and what was its intended purpose.
```

Comment:
Change the last sentence to "Bitcoin's first owners knew well why Bitcoin was invented and for what purpose it was intended."

Reply:
Remove comma after "Shelling out: The Origins of Money"